ArcGISRuntimeSDKQt_QMLSamples/Scenes/TerrainExaggeration/README.md

```diff
@@ -0,0 +1,31 @@
+# Terrain exaggeration
+
+This sample demonstrates how to XXXXX.
+This sample demonstrates ...       
+This is **why** you would do it this way ...
+
+![](screenshot.png)
```

Comment:
your screenshots need to be updated to a 500x500 screenshot of the sample as well